src/components/ha-icon-overflow-menu.ts

```diff
@@ -46,30 +47,32 @@ export class HaIconOverflowMenu extends LitElement {
                 slot="trigger"
               ></ha-icon-button>
 
-              ${this.items.map(
-                (item) => html`
-                  <mwc-list-item
-                    graphic="icon"
-                    .disabled=${item.disabled}
-                    @click=${item.action}
-                    class=${classMap({ warning: Boolean(item.warning) })}
-                  >
-                    <div slot="graphic">
-                      <ha-svg-icon
-                        class=${classMap({ warning: Boolean(item.warning) })}
-                        .path=${item.path}
-                      ></ha-svg-icon>
-                    </div>
-                    ${item.label}
-                  </mwc-list-item>
-                `
+              ${this.items.map((item) =>
+                item.divider
+                  ? html`<li divider role="separator"></li>`
+                  : html`<mwc-list-item
+                      graphic="icon"
+                      .disabled=${item.disabled}
+                      @click=${item.action}
+                      class=${classMap({ warning: Boolean(item.warning) })}
+                    >
+                      <div slot="graphic">
+                        <ha-svg-icon
+                          class=${classMap({ warning: Boolean(item.warning) })}
+                          .path=${item.path}
+                        ></ha-svg-icon>
+                      </div>
+                      ${item.label}
+                    </mwc-list-item> `
               )}
             </ha-button-menu>`
         : html`
             <!-- Icon representation for big screens -->
             ${this.items.map((item) =>
               item.narrowOnly
                 ? ""
+                : item.divider
+                ? html`<li divider role="separator"></li>`
```

Comment:
Does this work? As it is not a list or something...

Reply:
I had tested via Chome dev tools both in desktop mode and in mobile mode and in both scenarios I see the separator in the menu.

Reply:
you have to set narrow explicitly, only backup panel uses it now I think

Reply:
It is already in narrow mode, so the non-narrow one is actually the issue: https://github.com/spacegaier/frontend/blob/ec257710ffc4298ba9a2f270b39b9900ee0a1d31/src/panels/config/automation/ha-automation-picker.ts#L192

Will push a fix in a minute.